Bixal/move.mil
# Blue intro text box - theming & config

Resolution of Issue #62 

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Adds config for a new basic html wysiwyg "intro text" field on the basic page content type and removes the body field from display (didn't remove body entirely but we always can later)
- Adds styles for that field's content creating the blue box around it
- Unchecks the css and js aggregation boxes, because having to manually uncheck them every time I worked on a new ticket was starting to annoy me
- Converts a spacing formula that I suspect will need to be frequently used to a sass mixin

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Run `make cim` to import config
1. run `npm build` from theme folder to aggregate sass changes
1. create/edit a basic page item and add text into the new "intro text" field. Save changes.
1. The blue box should appear just below the page title (or admin tabs if logged in)

## Screenshots

Blue box on local:
<img width="1086" alt="screen shot 2018-04-09 at 10 29 59 am" src="https://user-images.githubusercontent.com/29130580/38504067-bb359fc2-3be1-11e8-819e-a934506359f3.png">
